src/modules/dashboard/components/RecommendedSnap.tsx

```diff
@@ -0,0 +1,91 @@
+// Third-party libraries
+import { useNavigate } from 'react-router-dom';
+import { css } from 'styled-components';
+
+//Constants
+import { RecommendedChatList } from '../Dashboard.constants';
+
+// Components
+import { Box, Text, Button } from 'blocks';
+import { ChatItem } from './ChatItem';
+import Metamask from 'blocks/illustrations/components/Metamask';
+
+const RecommendedSnap = () => {
+  const navigate = useNavigate();
+
+  return (
+    <Box
+      display="flex"
+      flexDirection="column"
+      width={{ ml: '100%', initial: '50%' }}
+      gap="s6"
+    >
+      <Text
+        color={{ light: 'gray-1000', dark: 'white' }}
+        variant="h4-bold"
+      >
+        Recommended Chats
+      </Text>
+      <Box
+        display="flex"
+        flexDirection="column"
+        gap="s4"
+        borderRadius="var(--r6)"
+        border={{ light: '1px solid gray-200', dark: '1px solid gray-800' }}
+        padding="s4"
+      >
+        {RecommendedChatList.map((item, index) => {
+          return (
+            <ChatItem
+              key={index}
+              chat={item}
+            />
+          );
+        })}
+      </Box>
```

Comment:
This 
```suggestion
        {recommendedChatList.map((item, index) => {
          return (
            <RecommendedChatListItem
              key={index}
              chat={item}
            />
          );
        })}
      </Box>
```
@mishramonalisha76